server/test/services/export/JsonExporterTest.java

```diff
@@ -163,6 +181,313 @@ private void testApplicationTopLevelAnswers(
     resultAsserter.assertValueAtPath("$[" + resultIndex + "].submitter_email", "Applicant");
   }
 
+  // ADDRESS
+  @Test
+  public void export_whenAddressQuestionIsAnswered_valueIsInResponse() throws Exception {
+    createFakeQuestions();
+    createFakeProgram();
+    new FakeApplicationFiller()
+        .answerAddressQuestion("12345 E South St", "Apt 8i", "CityVille Township", "OR", "54321")
+        .submit();
+
+    JsonExporter exporter = instanceOf(JsonExporter.class);
+
+    String resultJsonString =
+        exporter
+            .export(
+                fakeProgram.getProgramDefinition(),
+                IdentifierBasedPaginationSpec.MAX_PAGE_SIZE_SPEC_LONG,
+                SubmittedApplicationFilter.EMPTY)
+            .getLeft();
+    ResultAsserter resultAsserter = new ResultAsserter(resultJsonString);
+
+    resultAsserter.assertValueAtApplicationPath(".applicant_address.street", "12345 E South St");
+    resultAsserter.assertValueAtApplicationPath(".applicant_address.line2", "Apt 8i");
+    resultAsserter.assertValueAtApplicationPath(".applicant_address.city", "CityVille Township");
+    resultAsserter.assertValueAtApplicationPath(".applicant_address.state", "OR");
+    resultAsserter.assertValueAtApplicationPath(".applicant_address.zip", "54321");
+  }
+
+  @Test
+  public void export_whenAddressQuestionIsNotAnswered_valueInResponseIsNull() throws Exception {
```

Comment:
Remove unthrown exceptions, here and elsewhere

Reply:
Done

Reply:
Still seeing it, did you push the change?